docs/src/tutorials/creating_geometry.md

```diff
@@ -0,0 +1,338 @@
+
+````@example creating_geometry
+crs = GFT.EPSG(4326)
+````
+
+Let's start by making a single `Point`.
+
+````@example creating_geometry
+point = GI.Point(0, 0)
+````
+
+Now, let's plot our point.
+
+````@example creating_geometry
+fig, ax, plt = plot(point)
+````
+
+Let's create a set of points, and have a bit more fun with plotting.
+
+````@example creating_geometry
+x = [-5, 0, 5, 0]
+y = [0, -5, 0, 5]
+points = GI.Point.(zip(x,y))
+plot!(ax, points; marker = '✈', markersize = 30)
+fig
+````
+
+Points can be combined into a single `MultiPoint` geometry. This time let's include information on CRS with the geometry making it a geospatial geometry. All that's needed is to include `; crs = crs` as a keyword argument when constucting the geometry.
+
+!!! note
+    It is good practice to only include CRS information with the highest-level geometry. Not doing so can bloat the memory footprint of the geometry. CRS information _can_ be included at the individual `Point` level but is discouraged.
+
+````@example creating_geometry
+x = [-5, -5, 5, 5]
+y = [-5, 5, 5, -5]
+multipoint = GI.MultiPoint(GI.Point.(zip(x, y)); crs)
+plot!(ax, multipoint; marker = '☁', markersize = 30)
+fig
+````
+
+Let's create a line between two points.
+
+````@example creating_geometry
+p1 = GI.Point.(-5, 0)
+p2 = GI.Point.(5, 0)
+line = GI.LineString([p1,p2]; crs)
+plot!(ax, line)
+fig
+````
+
+Now, let's create a line connecting multiple points (i.e. a `LineString`).
+This time we get a bit more fancy with point creation.
+
+````@example creating_geometry
+r = 2;
+k = 10;
+ϴ = 0:0.01:2pi
+x = r .* (k + 1) .* cos.(ϴ) .- r .* cos.((k + 1) .* ϴ)
+y = r .* (k + 1) .* sin.(ϴ) .- r .* sin.((k + 1) .* ϴ)
+lines = GI.LineString(GI.Point.(zip(x,y)); crs)
+plot!(ax, lines; linewidth = 3)
+fig
+````
+
+We can also create a single `LinearRing` trait, the building block of a polygon.
+A `LinearRing` is simply a `LineString` with the same beginning and endpoint,
+i.e., an arbitrary closed shape composed of point pairs.
+
+A `LinearRing` is composed of a series of points listed in clockwise order (i.e., winding order).
+I always think of a polygon as filled to the right of the lines as one progresses
+from point `n` to point `n+1`.
+
+````@example creating_geometry
+ring1 = GI.LinearRing(GI.getpoint(lines))
+````
+
+Now, let's make the `LinearRing` into a `Polygon`.
+
+````@example creating_geometry
+polygon1 = GI.Polygon([ring1]; crs)
+````
+
+Now, we can use GeometryOperations and CoordinateTransformations to shift `polygon1`
+vertically up, to avoid plotting over our earlier results.
+
+
+:::tabs
+
+== CoordinateTransformations
+
+````@example creating_geometry
+xoffset = 0.
+yoffset = 50.
+f = CoordinateTransformations.Translation(xoffset, yoffset)
+polygon1 = GO.transform(f, polygon1)
+plot!(polygon1)
+fig
+````
+
+== Direct manipulation
```

Comment:
Can you add some guidance on why one would choose Direct manipulation over CoordinateTransformations ? Having both is a bit confusing

Reply:
The main reason I wanted to show this is to show users that you can do this directly and don't have to rely on CoordinateTransformations.  

CoordinateTransformations as a package is also only particularly good for affine transformations.  So if you wanted to transform the coordinates using your own polar transform or something, it may not be immediately clear how.  Then again, at that point you've probably read the docs...

If it's too confusing we can remove it, but IMO it's nice to at least show the option.

Reply:
Given that this is suppose to be a gentle introduction, and is in now way verbose, I'm inclined to omit  `Direct manipulation` as it uses a `do` block which I find the absolutely worst syntax for intuitive understanding... but if you object let's publish